monkey/infection_monkey/network/mysqlfinger.py

```diff
@@ -2,16 +2,17 @@
 import socket
 
 import infection_monkey.config
+import infection_monkey.network.HostFinger
```

Comment:
why not `from infection_monkey.network.HostFinger import HostFinger`?

Reply:
@danielguardicore Notice that you need to fix this across all the PR (it happens a few times)